src/v1/components/data-display/UserProfile/UserProfile.tsx

```diff
@@ -7,6 +7,7 @@ import MUIMenu from "@mui/material/Menu";
 import { H2 } from "../Text/Text";
 import UserIcon from "../Icons/UserIcon";
 import DownArrowIcon from "../FontAwesomeIcons/DownArrowIcon";
+import "./user-profile.css";
```

Comment:
Ash is keen on not using css in any of the v1 components. You will need to convert this to js styling. Also just checking if Justin is aware of the changes made here.

Reply:
JS styling now used.

Changes shown to @Justinki50.